sentinel/src/lib/schedules.js

```diff
@@ -122,6 +127,9 @@ module.exports = {
                 }
                 // don't schedule messages in the past or empty messages
                 if (due < now || !message) {
+                    if (schedule.start_mid_group === false) {
```

Comment:
This means the default (unconfigured) behaviour is to start mid group, right? This makes sense to me because it means old configurations don't change. However I think most of the time it's a bug if the messages start mid-group. From the issue:

> If the first messages of a group are in the past we should not be sending the latter messages of the group. Instead, the first scheduled messages that should be sent are the first messages of the next group.

This suggests it shouldn't even be configurable.

@abbyad Do you think this should be...

a) hardcoded to never start mid group,
b) configurable defaulting to not starting mid-group possibly breaking existing configurations,
c) configurable defaulting to starting mid-group, which is backwards compatible but probably a bad default, or
d) configurable defaulting to not starting mid-group with a migration to set `start_mid_group: true` on existing configurations

I'm leaning towards (b) because I think that's the expected behaviour in almost all situations so it would essentially fix an existing bug, while allowing the unusual situations to be fixed in config.

Reply:
I agree, and think option B is best to have the desired behaviour as the default. It is unlikely that it would break an existing config, but if it does it is easy enough to add `start_mid_group: true` to that schedule.

Reply:
Ok. @wtekeu I think you can change the line to...

```js
if (!schedule.start_mid_group) {
```

Then add the documentation of the feature and it'll be good to go!